modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -3,1080 +3,1093 @@
   "description": "Claims API 526 Schema",
```

Comment:
Just wanted to update this.  v2.rb still has a change, related to setting the request file to be shown.  Otherwise, these other changes were all rolled back in my latest commit.